runtime/compilers/rillv1/init.go

```diff
@@ -18,7 +18,8 @@ func IsInit(ctx context.Context, repo drivers.RepoStore, instanceID string) bool
 func InitEmpty(ctx context.Context, repo drivers.RepoStore, instanceID, title string) error {
 	mockUsersInfo := "# These are example mock users to test your security policies.\n# For more information, see the documentation: https://docs.rilldata.com/manage/security"
 	mockUsers := "mock_users:\n- email: john@yourcompany.com\n- email: jane@partnercompany.com"
-	rillYAML := fmt.Sprintf("compiler: %s\n\ntitle: %q\n\n%s\n\n%s", Version, title, mockUsersInfo, mockUsers)
+	ignoredPaths := "ignored_paths:\n- /.idea\n- /.vscode" // TODO: get a list from product
+	rillYAML := fmt.Sprintf("compiler: %s\n\ntitle: %q\n\n%s\n\n%s\n\n%s", Version, title, mockUsersInfo, mockUsers, ignoredPaths)
```

Comment:
What do you think about not adding it by default to avoid clutter? And just describing it in the docs. (Assuming our defaults are good enough for most people.)

Reply:
This was the ask from product. But if we have hardcoded paths then this might not be necessary. @mindspank Any comments?

Reply:
Hmm feels like with our hardcoded paths we don't need defaults in rill.yaml